Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -2105,6 +2105,11 @@ void DocumentLoader::startLoadingMainResource()
 
     Ref<DocumentLoader> protectedThis(*this);
 
+    if (m_request.url().protocolIsAbout() && !(m_request.url().isAboutBlank() || m_request.url().isAboutSrcDoc())) {
```

Comment:
I wonder if there is loading spec language that covers this stuff. Should we create a WPT test for this at some point?

Reply:
special schemes section does not mention `about:` in https://url.spec.whatwg.org/#url-miscellaneous
@annevk would know.

Reply:
https://fetch.spec.whatwg.org/#scheme-fetch defines that for fetching only "about:blank" should give you something. And navigation special cases `about:srcdoc` as far as I can tell (look at the caller of https://html.spec.whatwg.org/#create-navigation-params-from-a-srcdoc-resource). So this behavior is covered by the standard and correct.